mmdet/core/hook/yolox_mode_switch_hook.py

```diff
@@ -15,14 +15,20 @@ class YOLOXModeSwitchHook(Hook):
             training to close the data augmentation and switch to L1 loss.
             Default: 15.
        skip_type_keys (list[str], optional): Sequence of type string to be
-            skip pipeline. Default: ('Mosaic', 'RandomAffine', 'MixUp')
+            skip pipeline. Default: ('Mosaic', 'RandomAffine', 'MixUp').
+       is_multiprocess (bool): Determine whether it is a multi-process.
```

Comment:
It needs to be noted that this variable is related to the multi-process of dataloader. Because dist training is also a kind of multi-process, but when the workers of dataloader can also be set to 0.